docs/components/gantry/single-axis.md

```diff
@@ -7,11 +7,11 @@ description: "Configure a single-axis gantry."
 images: ["/icons/components/gantry.svg"]
 tags: ["gantry", "components"]
 aliases:
-    - "/components/gantry/oneaxis/"
-# SME: Rand
+    - "/components/gantry/singleaxis/"
```

Comment:
Please undo this change. This was correct as is. The page was called this before and therefore we need a redirect